authorship.yaml

```diff
@@ -1,2 +1,3 @@
 data: frida/frida.sha256
+vocalise_data: frida/vocalize_results/Output_6-28-2021 2-19-04 PM_2019A-XVector-Adaptable-20F-512-OCMS-29May2019_XVector - PLDA_all_edits.csv
```

Comment:
reference to a file which is not included

Reply:
name change done. the file is 300mb, i prefer not to checked it in.